### DISCUSSION TITLE
Parse error on start up the terminal (MacOS)

### DISCUSSION THREAD

**Discussion by esturniolo** [ohmyzsh/ohmyzsh#10695] [Help]
Hi everybody!!

I've OMZ in my Mac withouth any problems. A few days ago, as usually, it tries to update itself but it can't. The output says:
`/Users/MyUser/.oh-my-zsh/oh-my-zsh.sh:181: parse error near `<<<'`

I must work in that moment and I don't give any importance to that.

Days passed and now, everytime that I open my terminal I see the same legend: `parse error near <<<` and it never try to update again.

Any ideas?

Thanks in advacne!

**Comment by mcornella** [comment]
https://github.com/ohmyzsh/ohmyzsh/issues/10281
